Formula/diff-so-fancy.rb

```diff
@@ -1,17 +1,20 @@
 class DiffSoFancy < Formula
   desc "Good-lookin' diffs with diff-highlight and more"
   homepage "https://github.com/so-fancy/diff-so-fancy"
-  url "https://github.com/so-fancy/diff-so-fancy/archive/v0.6.3.tar.gz"
-  sha256 "b56213f08e5f1de1b0529d1a9a62024913ad2f12e043f9818f8cfcf00bed55c4"
+  url "https://github.com/so-fancy/diff-so-fancy/archive/v0.7.1.tar.gz"
+  sha256 "dcf795df0b398f393215d78679f34427e65aa263be3f4016e7e618706b1b4049"
 
   bottle :unneeded
 
   def install
-    prefix.install Dir["third_party", "libs", "diff-so-fancy"]
+    inreplace "diff-so-fancy", "/lib/", "/libs/"
+    prefix.install Dir["third_party", "diff-so-fancy"]
+    prefix.install "lib" => "libs"
```

Comment:
It probably belongs in `share` or `libexec`. `libs` wasn't standard to begin with, and upstream got rid of it. They probably switched to `lib` since there's only one, but it doesn't make any more sense than `libs` did. It might be worth filing an issue about, but either way, perpetuating `libs` seems like a bad idea.